### ISSUE TITLE
contacting server problem 

### ISSUE THREAD

**Issue by jgowtham09** [lhchavez/gtalkchat#171]
Hi,
 unable to contact server error whenever i login,
 even though using 3G . Please fix the bug... Its really required....

**Comment by ghvbk** [comment]
Yes not happening in both Windows 8 and my current 8.1 version. It is the only thing now I regret of switching over to Windows OS from Android.Needs to be solved ASAP

**Comment by goody91** [comment]
Whenever I login unable to contact server error.what is it.plz solve it as soon as posible.
